src/main/java/edu/harvard/iq/dataverse/externaltools/ExternalTool.java

```diff
@@ -328,4 +330,21 @@ public String toString() {
         }
     }
 
+    public String getDescriptionLang() {
+        if (this.getDisplayName().equals(BundleUtil.getStringFromDefaultBundleEng("externaltools.dct.displayname"))) {
+            return (BundleUtil.getStringFromBundle("externaltools.dct.description"));
+        } else {
+            return this.getDescription();
+        }
+    }
+
+    public String getDisplayNameLang() {
+        if (this.getDisplayName().equals(BundleUtil.getStringFromDefaultBundleEng("externaltools.dct.displayname"))) {
+            return (BundleUtil.getStringFromBundle("externaltools.dct.displayname"));
+        } else {
+            return this.getDisplayName();
+        }
+    }
```

Comment:
Granted there's only one config tool I know of, but won't this call all config tools "Data Curation Tool"? Can the string name be parametrized per tool/by tool name?